app/assets/javascripts/mobile/mobile_drawer.js

```diff
@@ -1,24 +1,151 @@
-(function(){
+(function() {
   Diaspora.Mobile.Drawer = {
-    initialize: function(){
-      $("#all_aspects").bind("tap click", function(evt){
+    initialize: function() {
+      this.drawer = $("#drawer");
+      this.mainContent = $("#main");
+      this.dragZone = this.createDragZone();
+      this.mainContent.append(this.dragZone);
+
+      var self = this;
+      var dateLastEvent = Date.now();
+      var debouncingThreshold = 40;
+      var hammerOpts = {
```

Comment:
AFAIR we use the .eslintrc from the develop branch to make sure that pronto uses an up-to-date version even if the PR hasn't been rebased for some time. Since you changed the .eslintrc in this PR you can just ignore this comment.